rules/docker-base-stretch.mk

```diff
@@ -5,14 +5,15 @@ $(DOCKER_BASE_STRETCH)_PATH = $(DOCKERS_PATH)/docker-base-stretch
 $(DOCKER_BASE_STRETCH)_DEPENDS += $(SUPERVISOR)
 $(DOCKER_BASE_STRETCH)_DEPENDS += $(SOCAT)
 
-ifeq ($(INSTALL_DEBUG_TOOLS),y)
 GDB = gdb
 GDBSERVER = gdbserver
 VIM = vim
 OPENSSH = openssh-client
 SSHPASS = sshpass
 STRACE = strace
-$(DOCKER_BASE_STRETCH)_DBG_PACKAGES += $(GDB) $(GDBSERVER) $(VIM) $(OPENSSH) $(SSHPASS) $(STRACE)
+$(DOCKER_BASE_STRETCH)_DBG_IMAGE_PACKAGES += $(GDB) $(GDBSERVER) $(VIM) $(OPENSSH) $(SSHPASS) $(STRACE)
+ifeq ($(INSTALL_DEBUG_TOOLS),y)
```

Comment:
since we are install those debug packages in the DBG docker images, we likely do not need this option now.

Reply:
Eventually we would not need.  Below is my comments (copy/paste from commit comments):

"
Tweaked the changes to retain the existing behavior w.r.t INSTALL_DEBUG_TOOLS=y.

When this change ('building debug docker image transparently') is extended to all dockers, this flag would become redundant. Yet, there can be some test based use cases that rely on this flag.

Until after all the dockers gets their debug images by default and we switch all use cases of this flag to use the newly built debug images, we need to maintain the existing behavior.
"